opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseDrawerInteractor.java

```diff
@@ -65,4 +86,55 @@ public void run() {
         appExecutors.diskIO().execute(runnable);
 
     }
+
+    @Override
+    public void checkSynced() {
+
+        String syncQuery = String.format("SELECT %s FROM %s WHERE %s <> ?\n", SYNC_STATUS, CLIENT_TABLE, SYNC_STATUS) +
+                "UNION ALL\n" +
+                String.format("SELECT %s FROM %s WHERE %s <> ? AND %s <> ?\n", SYNC_STATUS, EVENT_TABLE, SYNC_STATUS, SYNC_STATUS) +
+                "UNION ALL\n" +
+                String.format("SELECT %s FROM %s WHERE %s <> ?\n", SYNC___STATUS, TASK_TABLE, SYNC___STATUS) +
+                "UNION ALL\n" +
+                String.format("SELECT %s FROM %s WHERE %s <> ?\n", SYNC___STATUS, STRUCTURE_TABLE, SYNC___STATUS);
+
+        Runnable runnable = new Runnable() {
+            @Override
+            public void run() {
+                Cursor syncCursor = null;
+                try
+                {
+                    syncCursor  = database.rawQuery(syncQuery, new String[]{SYNCED, SYNCED, TASK_UNPROCESSED, SYNCED, SYNCED});
```

Comment:
Use org.smartregister.repository.BaseRepository#TYPE_Synced instead of the `SYNCED` variable defined in this class. The variables in BaseRepository class are used everywhere in the application to refer to the synced state.

The same applies to `TASK_UNPROCESSED`